README.md

```diff
@@ -456,6 +456,8 @@ The following settings are optional and allow you to further configure your clus
   ephemeral-storage= "1Gi"
   ```
 
+* `settings.kubernetes.tls-certificate`: The base64 encoded content of an x509 certificate for Kubelet HTTPS communication.
+* `settings.kubernetes.tls-private-key`: The base64 encoded content of an x509 private key corresponding to the `tls-certificate` public certificate.
```

Comment:
I've said in the past that `settings.kubernetes` is misnamed, and should really be `settings.kubelet`.

If we pretend for a moment that the renaming has already happened, and that the "path" to this setting implicitly mentions `kubelet`, then I would prefer that these settings better match their function:
```suggestion
* `settings.kubernetes.server-certificate`: The base64 encoded content of an x509 certificate for the Kubelet web server, which is used for retrieving logs and executing commands.
* `settings.kubernetes.server-key`: The base64 encoded content of an x509 private key for the Kubelet web server.
```